src/makeInfinityAdder.js

```diff
@@ -7,6 +7,23 @@
 
 function makeInfinityAdder() {
   // write code here
+  const addRol = (out) => {
+    if (out !== undefined) {
+      const arr = (inner) => {
+        if (inner !== undefined) {
+          return addRol(out + inner);
+        } else {
+          return out;
+        }
+      };
+
+      return arr;
+    }
+
+    return 0;
+  };
+
+  return addRol;
```

Comment:
rename variables and use clear naming for them
and get rid of arrow function - you can store your sum in `makeInfinityAdder` directly